### ISSUE TITLE
Windows 10 - 21H1 - Unable to create a PwnCat Connection

### ISSUE THREAD

**Issue by jbaselierhwh** [calebstewart/pwncat#213]
## Bug Description
I tried a Powershell-based revshell - NetCat - MSFVenom Payload, PowerCat etc. No matter what method I use to create a reverse connection to my PwnCat listener (on Kali) I always get the following error (see screenshot). 
![error1](https://user-images.githubusercontent.com/78091974/139579696-97b1e795-d183-4ac7-8977-96fa7697437e.jpg)

## pwncat version
Provide the output of `pwncat --version` or a commit hash if working from
a development branch.
0.4.3
```
$ pwncat --version
0.4.3
```

## Target System (aka "victim")
Provide as much detail about the target host as possible. If this is a TryHackMe
Windows 10 - 21H1
Virtual Machine
No Antivirus
No Firewall

## Steps to Reproduce
1. See screenshot to start a listener and create a reverse shell (no matter what method I use)

## Expected Behavior
An interactive shell

**Comment by a-k-r-e-c-h** [comment]
Yes, I also get error while tried to get a reverse shell  from TryHackMe windows machines. Every time ot gives an error. I hope developers will fix it soon.Best of luck💗✌️
